docs/contributing/first-time.md

```diff
@@ -106,6 +133,20 @@ After you have satisfied the above steps and have clear direction on how to proc
 
 ### Set up your environment
 
+As a member of the Plone Contributors Team, you do not have write access to push commits to GitHub repositories under the Plone organization.
```

Comment:
Yes, this is the case for first-time contributors. It was a severe security issue, and created a lot of other problems.

When GitHub was friendly, and we used the paper-based Plone Contributor Agreement, we tossed everyone into the [Developers Team](https://github.com/orgs/plone/teams/developers). Then we got hammered by Hacktoberfest and GSoC drive by users, so we created an intermediary [Contributors Team](https://github.com/orgs/plone/teams/contributors).

Details of this decision are at https://community.plone.org/t/gsoc-students-please-focus-on-your-application-restrictions-on-developer-team-access-on-github/17041

It's still not perfect, because we lack the full GitHub Enterprise plan, but it is the best we can manage for an open source project plan on GitHub.